xml/System.Data.Metadata.Edm/ObjectItemCollection.xml

```diff
@@ -303,10 +303,10 @@
       </Parameters>
       <Docs>
         <param name="objectSpaceType">A <see cref="T:System.Data.Metadata.Edm.StructuralType" /> that represents the object space type.</param>
-        <param name="clrType">The CLR type.</param>
-        <summary>Returns a CLR type corresponding to the <see cref="T:System.Data.Metadata.Edm.StructuralType" /> supplied by the objectSpaceType parameter.</summary>
+        <param name="clrType">When this method returns, contains the CLR enum type of the OSpace argument.</param>
+        <returns><see langword="true" /> if the CLR type can be obtained; otherwise, <see langword="false" />. If the DataSpace property of the parameter is not <see cref="F:System.Data.Metadata.Edm.DataSpace.OSpace" />, the method returns <see langword="false" /> and sets the <paramref name="clrType" /> parameter to <see langword="null" />.</returns>
         <returns>
-          <see langword="true" /> if there is a type that matches the search criteria; otherwise, <see langword="false" />.</returns>
+          <see langword="true" /> if there is a type that matches the search criteria; otherwise, <see langword="false" />. If the DataSpace property of the parameter is not <see cref="F:System.Data.Metadata.Edm.DataSpace.OSpace" />, the method returns <see langword="false" /> and sets the <paramref name="clrType" /> parameter to <see langword="null" />.</returns>
```

Comment:
It looks like the `<returns>` element is duplicated.

Reply:
Good catch!